crates/cxx-qt-lib/src/gui/qfont.rs

```diff
@@ -89,10 +91,28 @@ mod ffi {
         #[rust_name = "default_family"]
         fn defaultFamily(self: &QFont) -> QString;
 
+        /// Returns true if a window system font exactly matching
+        /// the settings of this font is available.
+        #[rust_name = "exact_match"]
+        fn exactMatch(self: &QFont) -> bool;
+
+        /// Returns the requested font family name, i.e. the name
+        /// set in the constructor or the last setFont() call.
+        fn family(self: &QFont) -> QString;
```

Comment:
At least for Qt 6, the documentation states that this is always the first family in the `families` call, which may be an empty array.

Maybe we should have this function return an `Option<QString>` and just be implemented in Rust by calling `families.first()`?

Reply:
If you do this refactor, you can also keep this method and set `#[rust_name="family_or_default"]`, as we have done with most other such APIs